activation/atxdb.go

```diff
@@ -775,14 +778,14 @@ func (db *DB) HandleAtxData(data []byte, syncer service.Fetcher) error {
 	}
 
 	if err := syncer.GetPoetProof(atx.GetPoetProofRef()); err != nil {
-		return fmt.Errorf("received atx (%v) with syntactically invalid or missing PoET proof (%x): %v",
-			atx.ShortString(), atx.GetShortPoetProofRef(), err)
+		return fmt.Errorf("received ATX (%v) with syntactically invalid or missing PoET proof (%x): %v",
```

Comment:
I've seen a PR from @lrettig where he changes all to lowercase `atx` , we probably need to decide on a convention..

Reply:
yes, I prefer all lowercase, it makes searching/grepping through logs easier - definitely lmk if you guys disagree